google/cloud/bigtable/examples/bigtable_samples_instance_admin.cc

```diff
@@ -243,6 +287,240 @@ void DeleteCluster(google::cloud::bigtable::InstanceAdmin instance_admin,
 }
 //! [delete cluster]
 
+//! [run instance operations]
+void RunInstanceOperations(
+    google::cloud::bigtable::InstanceAdmin instance_admin, int argc,
+    char* argv[]) {
+  if (argc != 4) {
+    throw Usage{"run: <project-id> <instance-id> <cluster-id> <zone>"};
+  }
+  google::cloud::bigtable::InstanceId instance_id(ConsumeArg(argc, argv));
+  google::cloud::bigtable::ClusterId cluster_id(ConsumeArg(argc, argv));
+  std::string const zone = ConsumeArg(argc, argv);
+
+  // Create production instance
+  google::cloud::bigtable::DisplayName display_name("Put description here");
+  auto cluster_config = google::cloud::bigtable::ClusterConfig(
+      zone, 3, google::cloud::bigtable::ClusterConfig::HDD);
+  google::cloud::bigtable::InstanceConfig config(
+      google::cloud::bigtable::InstanceId(instance_id), display_name,
+      {{cluster_id.get(), cluster_config}});
+  config.set_type(google::cloud::bigtable::InstanceConfig::PRODUCTION);
+
+  std::cout << std::endl << "Creating a PRODUCTION Instance: ";
```

Comment:
In this case is the better (and customary) to use:

```C++
  std:: cout << "\nCreating a PRODUCTION Instance: ";
```

`std::endl` flushes the iostream in addition to printing a newline.  Same comment elsewhere.

Reply:
fixed.